### ISSUE TITLE
IBus not running message is not very informative

### ISSUE THREAD

**Issue by Loreno10** [elementary/switchboard-plug-keyboard#360]
I think it would make sense to put some more information on this view:
![image](https://user-images.githubusercontent.com/10273406/116809247-57d71400-ab3d-11eb-8d87-0fbbd8e93cb1.png)

The "IBus is not running" message does not say a lot to a typical user. Maybe there could be some more explanation of what it actually is or some link to an explanation on the web (preferably to eOS docs?)

**Comment by jeremypw** [comment]
Something like "Input Methods are not currently enabled" perhaps?  The user does not need to know the name of the backend software.

The button could be "Enable Input Methods".

The secondary text is hardly necessary, but information like "Input Methods are needed to type some languages, such as Chinese, Japanese and Korean"

**Comment by jeremypw** [comment]
Another solution would be to simply startup the daemon when the tab is accessed if it is not already running and, if on leaving there are no input methods set, turn it off again.

**Comment by Loreno10** [comment]
Exactly, although I think there should be some small “additional info” button, which informs more advanced users that it actually is IBus
